mteb/evaluation/evaluators/RetrievalEvaluator.py

```diff
@@ -112,21 +100,12 @@ def search(
         corpus: dict[str, dict[str, str]],
         queries: dict[str, str | list[str]],
         top_k: int,
-        score_function: str,
```

Comment:
breaking? should we move to v2.0.0?

Reply:
I don't think this is very breaking because, in the implementation from the main branch, `score_function` can only be passed with `evaluator.run`, and I doubt many users have specified this.